server/controllers/authorController.js

```diff
@@ -61,6 +62,51 @@ class AuthorController {
     
     return util.successStatus(res, 200, 'Author deleted successfully');
   }
+
+  static async favouriteAnAuthor(req, res) {
```

Comment:
THought we already merged this?

Reply:
I dont know what happened there
I just rebased from develop